WooCommerce/Classes/Tools/Zendesk/ZendeskManager.swift

```diff
@@ -826,12 +890,18 @@ private extension ZendeskManager {
         static let nameFieldCharacterLimit = 50
         static let sourcePlatform = "mobile_-_woo_ios"
         static let subcategory = "WooCommerce Mobile Apps"
+        static let paymentsCategory = "support"
+        static let paymentsSubcategory = "payment"
+        static let paymentsProduct = "woocommerce_payments"
```

Comment:
This is fine for now, but we'll want to support `woocommerce_stripe` as a product in the future too

Reply:
This is a requirement issued by the HEs that hav setup the queues and the funnel to receive these tickets and move them to the approbate queue 🤷🏼